lib/MusicBrainz/Script/JSONDump.pm

```diff
@@ -71,10 +71,19 @@ has output_dir => (
 
 with 'MusicBrainz::Script::Role::TestCacheNamespace';
 
-our $TMP_EXPORT_DIR = tempdir(
-    'json-dump-XXXXXX',
-    DIR => tmpdir(),
-    CLEANUP => 0,
+has tmp_export_dir => (
+    is => 'ro',
+    isa => 'Str',
+    lazy => 1,
+    default => sub {
+      my @output_path = splitdir($self->output_dir);
+      pop @output_path;
```

Comment:
Maybe we can stick with `tmpdir()` by default but allow overriding it via an argument to the script?  If I use `~/musicbrainz-server` as the output dir while testing, I'm not sure I want it using my home dir as a temp dir and accidentally deleting stuff there.  In other cases there might not be permission to access `../`.

Reply:
It seems less clunky. Added option in commit <https://github.com/metabrainz/musicbrainz-server/pull/2350/commits/9ba591e2802857fb0b1ffc542b7e78473b043d0d>.